README.md

```diff
@@ -1 +1 @@
-# data-sync
\ No newline at end of file
+# data-sync
```

Comment:
Add some description about the repo/application responsibility and also add build instruction to compile this repo.